Core/src/Material/AccumulatedSurfaceMaterial.cpp

```diff
@@ -61,7 +61,9 @@ void Acts::AccumulatedSurfaceMaterial::trackAverage(const Vector3& gp,
                                                     bool emptyHit) {
   if (m_binUtility.dimensions() == 0) {
     m_accumulatedMaterial[0][0].trackAverage();
```

Comment:
I am pretty sure that it should be `.trackAverage(emptyHit);` else it won't do anything.

Reply:
Thanks! Done.